Cargo.toml

```diff
@@ -56,3 +56,6 @@ search-cache=["cachedirtree"]
 transcoding-cache=["simple-file-cache", "bytes"]
 libavformat = ["media_info"]
 
+[[bin]]
```

Comment:
fine, but as this is a default it probably is not necessary

Reply:
Added this for `cargo fetch`, which was failing otherwise if the `src/main.rs` file is missing.

Reply:
Interesting - learning new things every day :-)